i18n/en.json

```diff
@@ -505,7 +509,7 @@
   "admin.image.amazonS3EndpointDescription": "Hostname of your S3 Compatible Storage provider. Defaults to `s3.amazonaws.com`.",
   "admin.image.amazonS3EndpointExample": "E.g.: \"s3.amazonaws.com\"",
   "admin.image.amazonS3EndpointTitle": "Amazon S3 Endpoint:",
-  "admin.image.amazonS3IdDescription": "Obtain this credential from your Amazon EC2 administrator.",
+  "admin.image.amazonS3IdDescription": "(Optional) Only required if you do not want to authenticate to S3 using an IAM role. Enter the Access Key ID provided by your Amazon EC2 administrator.",
```

Comment:
Propose `IAM role` in the help text links to https://about.mattermost.com/default-iam-role if clicked.

Reply:
done